caom2repo/caom2repo/core.py

```diff
@@ -95,7 +95,9 @@
 
 BATCH_SIZE = int(10000)
 
-CAOM2REPO_OBS_CAPABILITY_ID =\
+CURRENT_CAOM2REPO_OBS_CAPABILITY_ID = \
+    'vos://cadc.nrc.ca~vospace/CADC/std/CAOM2Repository#obs-1.2'
+PREVIOUS_CAOM2REPO_OBS_CAPABILITY_ID =\
     'vos://cadc.nrc.ca~vospace/CADC/std/CAOM2Repository#obs-1.1'
```

Comment:
These should be the keys to a dictionary. The values would be CAOM2 namespaces from `caom2.obs_reader_writer` ex for `{'...obs-1.1': caom2.obs_reader_writer.CAOM23_NAMESPACE, `..obs-1.2: caom2.obs_reader_writer.CAOM24_NAMESPACE`}
The values of the dictionary will be used when instantiating the ObservationWriter in order to send the correct version of the document to the repo.

Reply:
Changed to something similar.